routes/api/route_helpers/game/getPlayedGame/getPlayedGameCallback.js

```diff
@@ -0,0 +1,29 @@
+const Game = require('../../../../../models/Game');
+const Word = require('../../../../../models/Word');
+
+const jwtDecode = require('../../jwtDecode');
+
+module.exports = (req, res) => {
+  const { gameId } = req.params;
+  const jwt = req.headers.authorization.split(' ')[1];
+  const { id: userId } = jwtDecode(jwt, res);
+
+  Game.findById(gameId).then(async (game) => {
+    if (game.user !== userId) res.status(401);
+
+    const sentWordIds = game.wordsSent;
+
+    const sentWords = await Promise.all(
+      sentWordIds.map((id) =>
+        Word.findById(id, { answer: 1, clue: 1, difficulty: 1 }).then(
```

Comment:
Ideally we would do some kind of aggregation and send only one request to the DB but I can't figure that part out right now